systemd/systemd.go

```diff
@@ -117,6 +117,38 @@ func MockSystemctl(f func(args ...string) ([]byte, error)) func() {
 	}
 }
 
+type ServiceState struct {
+	ActiveState   string
+	UnitFileState string
+}
+
+// If `cmd` is the command issued by systemd.Status(), this function returns
+// the output to be produced by the command so that the queried services will
+// appear having the ActiveState and UnitFileState according to the data
+// passed in the `states` map.
+func MockAllUnitsActiveOutput(cmd []string, states map[string]ServiceState) []byte {
```

Comment:
this function isn't quite a "mock" in the sense it is just a helper for writing mocks of systemctl, maybe something like:

```suggestion
func HandleMockAllUnitsActiveOutput(cmd []string, states map[string]ServiceState) []byte {
```

or something else that include mock in the name but not as the first word to indicate it is not primarily mocking anything.

Also this should use `osutil.MustBeTestBinary` to make sure it only gets used from tests.

Reply:
Done